tests/magick-image/remove-profile.spec.ts

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright Dirk Lemstra https://github.com/dlemstra/magick-wasm.
+// Licensed under the Apache License, Version 2.0.
+
+import { TestImages } from '../test-images';
+
+describe('MagickImage#removeProfile', () => {
+    it('should remove the profile from the image', () => {
+
```

Comment:
Can you remove the three empty lines after the braces? Not a big deal but it's more the style of the rest of the project.

Reply:
Yes, sure